### DISCUSSION TITLE
Radio Buoy item for radio/voice range extension

### DISCUSSION THREAD

**Discussion by Lexicality** [FakeFishGames/Barotrauma#11857] [Ideas and suggestions]
It would be good to have a neutrally buoyant item you could leave a trail of behind you while on a trip to extend the range of radio communications. (Like glowsticks!)

This would also open the opportunity for things like alien ruins blocking all radio out of them unless you leave one in the entrance and/or the electromagnetic interference they generate attracting creatures.

**Comment by Likvidus** [comment]
Good idea. Sounds like a small part of a big radio overhaul that i've been dreaming of for a long time.
